components/ui/development/set-hostname.yaml

```diff
@@ -0,0 +1,4 @@
+---
+- op: add
+  path: /spec/host
+  value: @TBA
```

Comment:
Kustomize build is failing because of the "@"
```suggestion
  value: TBA
```

Reply:
I added quotes. I want to keep the @ sign since it would be useful later when adjusting the dev scripts.

Reply:
You forgot to push the update?

Reply:
I pushed the update now